test/benchmark.js

```diff
@@ -65,6 +68,11 @@ urls.forEach(function(url) {
             results[url].sha1 = data.toString("hex");
             done();
         });
+        uglifyjs.stdout.pipe(zlib.createGzip({
+            level: zlib.Z_BEST_COMPRESSION
```

Comment:
> Don't web servers generally use default gzip compression?

For dynamic content, probably.

Static content are not compressed on the fly, so they tend to spend more effort in squeezing bigger savings.

In fact, there is `zopfli` which is crazy slow but produce `gzip`-compatible output - though I don't think there is widespread adoption of that one.